client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -0,0 +1,124 @@
+require('./toolBar.less');
+const React = require('react');
+const { useState, useEffect } = React;
+const _ = require('lodash')
+
+const MAX_ZOOM = 300;
+const MIN_ZOOM = 10;
+
+const ToolBar = ({ onZoomChange, currentPage, onPageChange, totalPages })=>{
+
+	const [zoomLevel, setZoomLevel] = useState(100);
+	const [pageInput, setPageInput] = useState(currentPage);
```

Comment:
`zoomLevel` is not named `zoomInput`, even though it is tracking a similar value in the toolbar. Can we rename `pageInput` to something else more fitting?

Reply:
Regarding your edit:    since there is a controlled text input, I can't just set it's `value` to `currentPage`, right?  Because then I wouldn't be able to change the text inside the box?

Reply:
Should be able to still edit the textbox. It would display the inherited `currentPage` to start, and as soon as you enter a new value, or scroll up and down, a new `currentPage` would filter down and show the updated number.